working-groups/community_meeting/26-01-2023.md

```diff
@@ -0,0 +1,47 @@
+# 26-01-2023 [community]  
+## Agenda - Open to Discussion
+- GSoC & GSoD participation
+- Publications about moja global projects
+- Participation in conferences/meetups
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
```

Comment:
Could you please change the technology discussed from Vuejs to "FLINT, FLINT.Cloud and more"